plugins/win-dshow/win-dshow.cpp

```diff
@@ -943,7 +931,11 @@ bool DShowInput::UpdateAudioConfig(obs_data_t *settings)
 			audioConfig.useVideoDevice ? "yes" : "no");
 
 	if (!audioConfig.useVideoDevice)
-		blog(LOG_INFO, "\taudio device: %s", (const char*)name_utf8);
+	{
```

Comment:
Style:  This curly brace should be at the end of the preceding line, placed after the if conditions and a single space.